ape-config.yaml

```diff
@@ -5,12 +5,29 @@ plugins:
     version: 0.7.0
 
 dependencies:
-  - name: smartcontractkit
-    github: smartcontractkit/chainlink-brownie-contracts
-    version: 0.8.0
+  # - name: openzeppelin
+  #   github: eth-optimism/contracts
+  #   ref: 4.3.3
+
+  # - name: openzeppelin-contracts-upgradeable
+  #   gituhb: openzeppelin/contracts-upgradeable-4.7.3
+    
+  # - name: ethoptimism
+  #   github: eth-optimism/contracts
+  #   ref: 0.5.21
+
+  # - name: smartcontractkit
```

Comment:
i think there is a better ape version of this dependency i recommend using.
Else, you will have to uncomment and fix all the peer dependencies above. I didnt go through and do all that for my own time's sake, but following the strategy of the other comment, you should be able to achieve that.